src/components/CustomCollection/CustomBookmark/CustomBookmark.test.tsx

```diff
@@ -27,51 +27,57 @@ describe('CustomBookmark component', () => {
     jest.clearAllMocks()
   })
 
-  it('renders a bookmark with an edit handler', () => {
+  it('renders a bookmark with an edit handler', async () => {
     renderWithModalRoot(
       <CustomBookmark bookmark={testBookmark} {...testHandlers} />
     )
+    await screen.findByText(testBookmark.label)
```

Comment:
The solution I found to fix the slew of act issues was to make sure and wait until the component is rendered before beginning any checks/tests.